README.md

```diff
@@ -54,6 +54,18 @@ To use these artifacts, you must also depend on `io.opentelemetry:opentelemetry-
 See [opentelemetry-java releases](https://github.com/open-telemetry/opentelemetry-java#releases) for
 more information.
 
+> Android Requirements
+> 
+> If you are using this on Android and your project's minSdk is lower than 26, you must enable
+> [desugaring](https://developer.android.com/studio/write/java8-support#library-desugaring).
+> See [#73](https://github.com/open-telemetry/opentelemetry-android/issues/73) for more information.
+> 
+> If your project's minSdk is lower than 24, in order to run instrumentation tests or run the app
+> built on debug, you must use AGP 8.3.0+ and set the `android.useFullClasspathForDexingTransform`
+> property in `gradle.properties` to `true` to ensure desugaring runs properly. For the full
+> context for this workaround, please see
+> [this issue](https://issuetracker.google.com/issues/230454566#comment18).
```

Comment:
<img width="941" alt="Screenshot 2024-05-06 at 10 20 04 AM" src="https://github.com/open-telemetry/semantic-conventions-java/assets/34418638/ead96c91-7e6b-420d-92f2-14512efda899">

This renders weird. Can you remove the `>` quoting and change `> Android Requirements` to `## Android Requirements`?